pkg/webhook/operatingsystemconfig/resources/templates/scripts/configure-rsyslog.tpl.sh

```diff
@@ -33,6 +33,10 @@ function configure_rsyslog() {
     systemctl enable rsyslog.service
   fi
 
+  if [[ ! -d {{ .rsyslogRelpQueueSpoolDir }} ]]; then
+    mkdir -p {{ .rsyslogRelpQueueSpoolDir }}
+  fi
```

Comment:
```suggestion
mkdir -p {{ .rsyslogRelpQueueSpoolDir }}
```
```
$ mkdir --help | grep -e "-p"
  -p, --parents     no error if existing, make parent directories as needed,
```
Same for `pkg/webhook/operatingsystemconfig/testdata/configure-rsyslog.sh`